distributions/aws/examples/vanilla/README.md

```diff
@@ -0,0 +1,370 @@
+# Deploying Kubeflow on EKS
+
+This guide describes how to deploy Kubeflow on AWS EKS.
+
+## Prerequisites
+
+This guide assumes that you have:
+
+1. Installed the following tools on the client machine
+    - [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) - A command line tool for interacting with AWS services.
+    - [eksctl](https://eksctl.io/introduction/#installation) - A command line tool for working with EKS clusters.
+    - [kubectl](https://kubernetes.io/docs/tasks/tools) - A command line tool for working with Kubernetes clusters.
+    - [yq](https://mikefarah.gitbook.io/yq) - A command line tool for YAML processing. (For Linux environments, use the [wget plain binary installation](https://mikefarah.gitbook.io/yq/#wget))
```

Comment:
Link doesn’t seem to be accurate and suggests going to GitHub page for wget https://github.com/mikefarah/yq/#wget